File: tools/bench/tasks/08_invoice.py
items = [
    {"name": "widget", "qty": 4, "price": 2.5},
    {"name": "gizmo", "qty": 2, "price": 7.25},
    {"name": "bolt", "qty": 10, "price": 0.1},
]
total = sum(it["qty"] * it["price"] for it in items)
print(total)
dearest = max(items, key=lambda it: it["qty"] * it["price"])
print(dearest["name"])

File: tools/bench/answers/go_haiku/s1/08_invoice.go
package main

import "fmt"

func main() {
	items := []struct {
		name  string
		qty   int
		price float64
	}{
		{"widget", 4, 2.5},
		{"gizmo", 2, 7.25},
		{"bolt", 10, 0.1},
	}
	grand := 0.0
	maxName := ""
	maxTotal := 0.0
	for _, item := range items {
		total := float64(item.qty) * item.price
		grand += total
		if total > maxTotal {
			maxTotal = total
			maxName = item.name
		}
	}
	fmt.Println(grand)
	fmt.Println(maxName)
}
